bgpd/bgp_route.c

```diff
@@ -3751,16 +3753,18 @@ void bgp_soft_reconfig_in(struct peer *peer, afi_t afi, safi_t safi)
 		bgp_soft_reconfig_table(peer, afi, safi, NULL, NULL);
 	else
 		for (rn = bgp_table_top(peer->bgp->rib[afi][safi]); rn;
-		     rn = bgp_route_next(rn))
-			if ((table = rn->info) != NULL) {
-				struct prefix_rd prd;
-				prd.family = AF_UNSPEC;
-				prd.prefixlen = 64;
-				memcpy(&prd.val, rn->p.u.val, 8);
+		     rn = bgp_route_next(rn)) {
+			table = bgp_table_get_node_info(rn);
+			if (!table)
+				continue;
```

Comment:
Would prefer to keep original code structure w/o continue.

Reply:
The new change is much more clearer and good coding practice since it aids with readability by doing an early check and continue with the rest of the main functionality indenting less.

Reply:
Either way this is essentially a white space change and should be put in its own commit (same pr is fine.)

Reply:
I've gone through and put the changes into their own commit.